### PR TITLE
feat(imagery): Import imagery config for upper-hutt_urban_2017_0.10m.

### PR DESCRIPTION
**Pull request by Wentao-Kuang**
https://basemaps.linz.govt.nz/?i=01FXGFB7J8MQJWRZ19R223H1TC&p=NZTM2000Quad&debug
https://basemaps.linz.govt.nz/?i=01FXGFBD4YZREYXYQJ7KQNVWEP&p=WebMercatorQuad&debug

### CODE REVIEW

**Comment by Wentao-Kuang** [comment]
Creating a testing pr to test the automatic commit config json format.

File: config/imagery/upper-hutt_urban_2017_0-10m-NZTM2000Quad.json [hunk 1]
```diff
@@ -1,94 +1,95 @@
...
+  "id": "01FXGFB7J8MQJWRZ19R223H1TC",
   "name": "upper-hutt_urban_2017_0-10m",
-  "id": "im_01F6P1XK2AVYDJ66NY2MKSEXW1",
-  "uri": "s3://linz-basemaps/2193/upper-hutt_urban_2017_0-10m/01F6P1XK2AVYDJ66NY2MKSEXW1",
-  "year": 2017,
   "projection": 2193,
-  "resolution": 100,
+  "uri": "s3://linz-basemaps/2193/upper-hutt_urban_2017_0-10m/01FXGFB7J8MQJWRZ19R223H1TC",
+  "year": 2017,
+  "resolution": 90,
```
Comment (blacha) [inline config/imagery/upper-hutt_urban_2017_0-10m-NZTM2000Quad.json:7]: @Wentao-Kuang this resolution looks odd?

Comment (Wentao-Kuang) [reply config/imagery/upper-hutt_urban_2017_0-10m-NZTM2000Quad.json:7]: The resolution get from `CogStacJob.output.quality`. Weird, looks different from the setting.

Comment (Wentao-Kuang) [reply config/imagery/upper-hutt_urban_2017_0-10m-NZTM2000Quad.json:7]: I might doing the wrong way, Should use `extractResolutionFromName`. I will update that in code.

Comment (blacha) [reply config/imagery/upper-hutt_urban_2017_0-10m-NZTM2000Quad.json:7]: `quality` is the JPEG/Webp quality right? so `extractResolutionFromName` would be better?

**Pull request closed without merging**